docker-compose.yml

```diff
@@ -9,8 +9,8 @@ services:
       MYSQL_USER: drupal
       MYSQL_PASSWORD: drupal
 #    command: --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci # The simple way to override the mariadb config.
-    volumes:
-      - ./docker-runtime/mariadb:/var/lib/mysql
```

Comment:
Ah that's much better than the `chcon` hack I learned to modify the files ....

Does that :Z flag cause problems when not running selinux? or is it safely ignored...